app/assets/javascripts/app/disabling_by_checkbox.coffee

```diff
@@ -0,0 +1,8 @@
+$ ->
+  $("[data-disables]").on "change", ->
```

Comment:
Code climate is complaining about `this` (aka `@`) without a fat arrow. If we make this a fat arrow, I think we would need to add something like `.on "change", (el) ->` and then `$(el)` (I may be wrong on the exact syntax).

Reply:
Yeah I kinda don't want to mess with the javascript right now if that's okay with you.